model-server/src/main/kotlin/org/modelix/model/server/handlers/RepositoriesManager.kt

```diff
@@ -200,18 +207,41 @@ class RepositoriesManager(val client: LocalModelClient) {
             ?: throw IllegalStateException("No version found for branch '${branch.branchName}' in repository '${branch.repositoryId}'")
     }
 
-    private val deltaCache = LRUMap<Pair<String, String?>, Lazy<Map<String, String>>>(10)
-    fun computeDelta(versionHash: String, baseVersionHash: String?): Map<String, String> {
+    private val deltaCache = LRUMap<Pair<String, String?>, SoftReference<Lazy<Map<String, String>>>>(10)
```

Comment:
Would it make sense to make the cache size configurable?

Reply:
This will soon also be replaced by a more efficient stream based implementation as part of [MODELIX-681](https://issues.modelix.org/issue/MODELIX-681/Explicit-partial-loading).